### ISSUE TITLE
Way to download other modules

### ISSUE THREAD

**Issue by hyperdriveguy** [hyperdriveguy/OpenGospel#10]
Download "Non-free" content (General Conference talks, Gospel Art, Handbooks, Magazines, and maybe Hymns). This should be separated from everything else. The problem is keeping a consistent format with the rest of the scriptures.

**Comment by hyperdriveguy** [comment]
Focus change. Now refocusing to have this be the primary goal right now. Goal is to refactor the code to basic support for a module based approach by this Saturday.

**Comment by hyperdriveguy** [comment]
Having a module store or repository will likely be the best way to go, after #21 is finished.

**Comment by jgibbons94** [comment]
If Gtk has a way to render PDFs we can use those.

**Comment by hyperdriveguy** [comment]
Because modules are focused around content delivery (particularly new, recent content from the church), this issue is likely best superseded by #29
